lib/features/children/add_member/repository/add_member_repository.dart

```diff
@@ -22,6 +22,7 @@ class AddMemberRepositoryImpl with AddMemberRepository {
     final profilesJsonList = members.map((member) => member.toJson()).toList();
     final body = {
       'profiles': profilesJsonList,
+      'allowanceType': 1,
```

Comment:
Can you check if this is also used by the generosity challenge code? If not then approved.